src/unix/internal.h

```diff
@@ -145,6 +145,12 @@ enum {
   UV_LOOP_BLOCK_SIGPROF = 1
 };
 
+/* flags of excluding ifaddr */
+enum {
```

Comment:
@saghul I used to use `typedef`. But I removed it later referred by https://github.com/libuv/libuv/pull/1375/commits/5b604f048190986204178cdfbdf48260cf862c85#r122405782.